tictactoe.py

```diff
@@ -1,26 +1,63 @@
+empty_squares = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+playerX_moves = []
```

Comment:
I really like the design choice of putting the player moves in lists like this, that opens up the possibility of building stuff like replay on top of your code later and would surely help if you need to debug :-)

Reply:
Do you mean replay as in playing again, or are you referring to something specific?

Reply:
I mean like a tv-replay of a show (if you missed the first airing) :-)